Automattic/wp-calypso
# A4A > Referrals: Implement initial Client UI

Closes https://github.com/Automattic/jetpack-genesis/issues/367

## Proposed Changes

This PR implements the initial client UI for A4A Referrals.

NOTE: 

- [Design](https://www.figma.com/design/fuufP6VNfZXYmvLsTRWRlG/Referrals?node-id=6711-71510&m=dev)

The lays down the foundational work for the client experience on A4A. This is the initial implementation, we might have to make some changes based on the API endpoint changes, 

## Testing Instructions

1. Checkout to the branch and start the server.
2. If you don't have existing referrals > Go to Marketplace > Toggle Refer products > Select product > Go to Checkout & Click the `Request payment from client` button
3. Now, get your agency ID & any client ID from the list of referrals and visit `/agency/{agencyId}/client/{clientId}`
4. Verify that you see the below page that includes 2 menu items > `Your Subscriptions` & `Payment Methods`


<img width="433" alt="Screenshot 2024-05-31 at 10 13 02 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/515d5d36-739e-407c-95b3-6f114d7ad85a">


## Pre-merge Checklist
